homeassistant/components/vulcan/calendar.py

```diff
@@ -111,11 +111,11 @@ async def async_get_events(
         event_list = []
         for item in events:
             event = CalendarEvent(
-                start=datetime.combine(item["date"], item["time"].from_).astimezone(
-                    ZoneInfo("Europe/Warsaw")
+                start=datetime.combine(item["date"], item["time"].from_).replace(
```

Comment:
`datetime.combine` takes a `tzinfo` argument that you can use directly rather than an additional call to replace.

Reply:
Thanks! I'm going to change it